certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -76,26 +75,26 @@ func (p *DeployableByOlmCheck) newOpenshiftEngine() (*openshiftengine.OpenshiftE
 
 func (p *DeployableByOlmCheck) operatorMetadata(bundleRef certification.ImageReference) (*OperatorData, error) {
 	// retrieve the operator metadata from bundle image
-	annotations, err := containerutil.GetAnnotationsFromBundle(bundleRef.ImageFSPath)
+	annotations, err := getAnnotationsFromBundle(bundleRef.ImageFSPath)
 
 	if err != nil {
 		log.Errorf("unable to get annotations.yaml from the bundle")
 		return nil, err
 	}
 
-	catalogImage, err := viperutil.GetString(indexImageKey)
+	catalogImage, err := getString(indexImageKey)
```

Comment:
This function call seems rather hard to follow to me. If I'm reading the code, I'd have a hard time deriving why I needed a function call to get a string, and this no longer has any context as to the Viper configuration side of the house. Is there any reason why we why we didn't just handle this config-key-empty-value case in-line?

Reply:
@samira-barouti would have to answer that. I'm not sure what the original intention of the function is.

Reply:
DeployableByOLM initially read many values from viper; that's why I had to manage the error handling outside the code. Right now, we only read `PFLT_INDEXIMAGE` which makes this code unnecessary.

Reply:
I inlined it. Thanks for the clarification.